### ISSUE TITLE
Sound applet: Last mousewheel step does not apply to all outputs.

### ISSUE THREAD

**Issue by ManuelSchneid3r** [linuxmint/Cinnamon#5013]
When I scroll the mousewheel on the applet icon the change applies to all outputs. However the last one to 100% does not. This results in a factor x which is less 1. If I do this a dozens (y) of times the volume of outputs converges to 0. ( 0 < x  < 1 => lim(y->infinity)(x^y) =0) :smile: 

A [video](https://youtu.be/YwuZqmysHmw) says more than thousands of words.

**Comment by rjanja** [comment]
I haven't been able to reproduce this. For me, the only slider on the "Applications" tab that changes is the "Output volume" at the very bottom. Individual application output volumes do not change. What version of Cinnamon are you running? 2.8.6 here.

**Comment by ManuelSchneid3r** [comment]
I'am on archlinux, hence last release

**Comment by Vahan86** [comment]
@ManuelSchneid3r, is this still an issue for you with Cinnamon 3?

**Comment by ManuelSchneid3r** [comment]
Yep [still an issue](https://www.youtube.com/watch?v=xNUi3u4NpvU&feature=youtu.be)

**Comment by ManuelSchneid3r** [comment]
Yes seems fixed, thank you.
